content/tools/reg.yaml

```diff
@@ -17,9 +17,9 @@ slides:
   link: "https://neurodata.io/talks/tools19.html"
 
 details:
-  - 'Neurodata''s open-source Python <a href="https://github.com/neurodata/image_lddmm_tensorflow">package</a> that performs affine and deformable (LDDMM) image registration.'
+  - 'Neurodata''s open-source Python <a href="https://github.com/neurodata/ardent">package</a> that performs affine and deformable (LDDMM) image registration.'
```

Comment:
Should be NeuroData to be consistent with rest of site and branding (I know this isn't something you did but we might as well change it)